src/common/main/kotlin/tech/thatgravyboat/skyblockapi/api/area/dungeon/DungeonFloor.kt

```diff
@@ -1,32 +1,37 @@
 package tech.thatgravyboat.skyblockapi.api.area.dungeon
 
+import tech.thatgravyboat.skyblockapi.api.area.isle.kuudra.KuudraTier
+import tech.thatgravyboat.skyblockapi.utils.extentions.valueOfOrNull
+
 enum class DungeonFloor(
     val bossName: String,
     val chatBossName: String = bossName,
     val floorNumber: Int,
+    val longName: String,
 ) {
-    E("The Watcher", 0),
+    E("The Watcher", 0, "The Catacombs Entrance"),
 
-    F1("Bonzo", 1),
-    F2("Scarf", 2),
-    F3("The Professor", 3),
-    F4("Thorn", 4),
-    F5("Livid", 5),
-    F6("Sadan", 6),
-    F7("Necron", "Maxor", 7),
+    F1("Bonzo", 1, "The Catacombs Floor I"),
+    F2("Scarf", 2, "The Catacombs Floor II"),
+    F3("The Professor", 3, "The Catacombs Floor III"),
+    F4("Thorn", 4, "The Catacombs Floor IV"),
+    F5("Livid", 5, "The Catacombs Floor V"),
+    F6("Sadan", 6, "The Catacombs Floor VI"),
+    F7("Necron", "Maxor", 7, "The Catacombs Floor VII"),
 
-    M1("Bonzo", 1),
-    M2("Scarf", 2),
-    M3("The Professor", 3),
-    M4("Thorn", 4),
-    M5("Livid", 5),
-    M6("Sadan", 6),
-    M7("Necron", "Maxor", 7),
+    M1("Bonzo", 1, "Master Mode The Catacombs Floor I"),
+    M2("Scarf", 2, "Master Mode The Catacombs Floor II"),
+    M3("The Professor", 3, "Master Mode The Catacombs Floor III"),
+    M4("Thorn", 4, "Master Mode The Catacombs Floor IV"),
+    M5("Livid", 5, "Master Mode The Catacombs Floor V"),
+    M6("Sadan", 6, "Master Mode The Catacombs Floor VI"),
+    M7("Necron", "Maxor", 7, "Master Mode The Catacombs Floor VII"),
     ;
 
-    constructor(bossName: String, floorNumber: Int) : this(bossName, bossName, floorNumber)
+    constructor(bossName: String, floorNumber: Int, longName: String) : this(bossName, bossName, floorNumber, longName)
 
     companion object {
-        fun getByName(name: String) = runCatching { DungeonFloor.valueOf(name) }.getOrNull()
+        fun getByName(name: String) = valueOfOrNull<KuudraTier>(name.lowercase())
```

Comment:
```suggestion
        fun getByName(name: String) = valueOfOrNull<DungeonFloor>(name.uppercase())
```

& remove the import

also it needs to be .uppercase since thats the enum name (also needs fixing in KuudraTier)